src/configs/OnboardingList.config.js

```diff
@@ -38,13 +40,45 @@ const signUpBluehostAcademy = () => {
   AnalyticsSdk.track("next_step", "next_step_bh_wp_academy_clicked", data);
   WordPressSdk.settings.put({ bluehost_academy_signup_clicked: true });
 };
+
+const updateSiteServers = (setWebServersUpdated) => {
+  AnalyticsSdk.track("next_step", "next_step_update_nameserver_clicked", data);
+  WordPressSdk.settings.put({ update_site_server_clicked: true }).then(() => {
+    WordPressSdk.settings.get();
+    setWebServersUpdated(true);
+  });
+};
+const updateSiteDomain = () => {
+  AnalyticsSdk.track("next_step", "next_step_connect_domain_clicked", data);
+};
+const updateMigrateViewGuide = () => {
+  AnalyticsSdk.track("next_step", "next_step_migrate_view_guide_clicked", data);
+};
+
+const updateStoreSetup = (setIsMigrationCompleted) => {
+  WordPressSdk.settings.put({ showMigrationSteps: false }).then(() => {
+    WordPressSdk.settings.get();
+    setIsMigrationCompleted(false);
+  });
+};
 const signUpYoastSEOAcademy = () => {
   WordPressSdk.settings.put({ yoast_seo_signup_status: true });
   AnalyticsSdk.track("next_step", "next_step_yoast_academy_clicked", data);
 };
 const brandName =
   (NewfoldRuntime?.sdk?.ecommerce?.brand_settings?.name).toLowerCase();
```

Comment:
Avoid using `NewfoldRuntime?.sdk`

Reply:
currently in newfoldRuntime, we're not getting ecommerce out of sdk, only once the ecommerce is out side of sdk, we can add this change

Reply:
Why can't we use `NewfoldRuntime?.plugin?.brand` we  can also skip `toLowerCase()` if we use that variable.

Reply:
We can't use plugin, becuase when bluehost changed it's brand like bluehost-us, hostgator_latam, at that time, this value wil not work

Reply:
As discussed on call we would go `split` way and add a `TODO`